INSTALL.md

```diff
@@ -370,4 +371,61 @@ In the "Trigger" section, check the ‘comments’ and ‘merge request events
 6. Test your installation by opening a merge request or commenting or a merge request using one of CodiumAI's commands.
 
 
+
+### Method 9: Run as a Bitbucket Pipeline
+
+
+You can use our pre-build Bitbucket-Pipeline docker image to run as Bitbucket-Pipeline.
+
+1. Add the following file in your repository bitbucket_pipelines.yml
+
+  pipelines:
+    pull-requests:
+      '**':
+        - step:
+            name: PR Agent Pipeline
+            caches:
+              - pip
+            image: python:3.8
+            services:
+              - docker
+            script:
+              - git clone https://github.com/Codium-ai/pr-agent.git
+              - cd pr-agent
+              - docker build -t bitbucket_runner:latest -f Dockerfile.bitbucket_pipeline .
+              - docker run -e OPENAI_API_KEY=$OPENAI_API_KEY -e BITBUCKET_BEARER_TOKEN=$BITBUCKET_BEARER_TOKEN -e BITBUCKET_PR_ID=$BITBUCKET_PR_ID -e BITBUCKET_REPO_SLUG=$BITBUCKET_REPO_SLUG -e BITBUCKET_WORKSPACE=$BITBUCKET_WORKSPACE bitbucket_runner:latest
```

Comment:
Please surround it code blocks, one for the yaml.
The environment variable should be in the yaml or run separately? if its separate, please use different code block.

Reply:
`bitbucket_runner:latest` should be inside the yaml file?